src/ports/sway/templates/README.md

```diff
@@ -0,0 +1,32 @@
+{{{ basic-heading }}}
+
+# Installation
+[`aura-theme.conf`](aura-theme.conf) has to be included in your Sway configuration. There are several options:
+
+* 1. Save the file (e.g. to `${XDG_CONFIG_HOME}/sway/aura-theme.conf`).
+  2. Include it in your Sway config (probably `${XDG_CONFIG_HOME}/sway/config`), by putting `include ${XDG_CONFIG_HOME}/sway/aura-theme.conf` somewhere.
+* 1. Copy the contents of [`aura-theme.conf`](aura-theme.conf).
+  2. Insert them somewhere into your Sway config (probably `${XDG_CONFIG_HOME}/sway/config`).
+* 1. Save [`aura-theme.conf`](aura-theme.conf) to your Sway drop-in directory (e.g. `${XDG_CONFIG_HOME}/sway/config.d` or `/etc/sway/config.d`).
+  2. Include the drop-in directory in you Sway config (probably `${XDG_CONFIG_HOME}/sway/config`), by putting `include ${XDG_CONFIG_HOME}/sway/config.d/*` somewhere.
+
+# Contributors
+<table>
+  <thead>
+    <tr>
+      <td valign="bottom"><p align="center">
+        <a href="https://github.com/Hendrikto">
+          <img src="https://github.com/Hendrikto.png?size=100" align="center" />
+        </a>
+      </p></td>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td><a href="https://github.com/Hendrikto">Hendrikto</a></td>
+    </tr>
+  </tbody>
+</table>
```

Comment:
Please, don't remove the **author-thead** and **author-tbody**, keep it as shown in [Publishing Ports doc](https://github.com/Hendrikto/aura-theme/blob/main/docs/PUBLISHING_PORTS.md):
```html
<table>
  <thead>
    <tr>
      <td valign="bottom"><p align="center">
        <a href="https://github.com/your-username">
          <img src="https://github.com/your-username.png?size=100" align="center" />
        </a>
      </p></td>
      {{{ author-thead }}}
    </tr>
  </thead>

  <tbody>
    <tr>
      <td><a href="https://github.com/your-username">Your name</a></td>
      {{{ author-tbody }}}
    </tr>
  </tbody>
</table>
```

Reply:
Fixed.